SRC/dragAndDrop.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable */
+
+import { listItems, swap } from './list.js';
+// ...............................
+export const addActionListner = () => {
+  const listElemenets = document.querySelectorAll('.list');
+  listElemenets.forEach((element) => {
+    element.addEventListener('dragstart', (e) => {
+      element.classList.add('ondrag');
+      e.dataTransfer.effectAllowed = 'move';
+      e.dataTransfer.setData('id', element.id);
+    });
+    element.addEventListener('drop', (e) => {
+      e.preventDefault();
+      e.dataTransfer.effectAllowed = 'move';
+      const fristId = e.dataTransfer.getData('id');
+      const secondId = element.id;
+      swap(fristId, secondId);
+    });
+    element.addEventListener('dragend', () => {
+      element.classList.remove('ondrag');
+    });
+    element.addEventListener('dragover', (e) => {
+      e.preventDefault();
+    });
+    const checked = element.children[0].children[0];
+    checked.addEventListener('change', () => {
+      const arrObject = listItems.find((ele) => ele.id === Number(element.id));
+      arrObject.completed = !arrObject.completed;
+      if (arrObject.completed) {
+        element.classList.add('checked');
+      } else {
+        element.classList.remove('checked');
+      }
```

Comment:
- [ ] You need to add a function that puts the updated array of items in the local storage, so the user gets the correct list values after the page reloads.